clients/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogSegmentMetadata.java

```diff
@@ -56,14 +56,14 @@
     private final int brokerId;
 
     /**
-     * Maximum timestamp in the segment
+     * Maximum timestamp in milli seconds in the segment
      */
-    private final long maxTimestamp;
+    private final long maxTimestampMillis;
```

Comment:
The more common way of using time units in variables seems to be to use the abbreviated suffix, https://github.com/apache/kafka/search?q=timeMs. In this case it would be `maxTimstampMs`. 

(same comment applies for other variables below).